lib/rubocop/cop/badge.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    # Identifier for all cops
+    class Badge
+      # Error raised when a badge parse fails
+      class InvalidBadge < Error
+        MSG = 'Invalid badge %<badge>p. ' \
```

Comment:
The `p` here means that I want `format` to call inspect. In other words `format("Value: %<name>s", name: 'John'.inspect)` is equivalent to `format("Value: %<name>p", name: 'John')`